### PR TITLE
Added IDE/editor page

### PR DESCRIPTION
**Pull request by hassila**
Useful information to find a decent editor/IDE.

### CODE REVIEW

**Review by weissi — APPROVED** [review]
thank you! Looks good to me!!

**Review by tomerd — APPROVED** [review]
this is great thanks. 

should we also mention:

* https://github.com/keith/swift.vim (or another vim plugin)
* https://atom.io/packages/ide-swift (or another atom plugin)

**Comment by hassila** [comment]
Sure, the more the merrier. One sec.

**Comment by hassila** [comment]
Added.

**Review by tomerd — APPROVED** [review]

File: ide-alternatives.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+[Xcode](https://developer.apple.com/xcode/ide/)
+
+[A plugin for VIM](https://github.com/keith/swift.vim)
+
+[A plugin for emacs](https://github.com/swift-emacs/swift-mode)
+
+[Atom IDE support](https://atom.io/packages/ide-swift)
+
+Also, the [SourceKit-LSP project](https://github.com/apple/sourcekit-lsp) has both an [extensive list of editors that supports it](https://github.com/apple/sourcekit-lsp/tree/main/Editors) and instructions on how to get completion in your favourite editor.
```
Comment (tomerd) [inline ide-alternatives.md:19]: Last comment: list them alphabetically?

**Review by ktoso — APPROVED** [review]
This is great! Very valuable to get people started :-)

+1 on alphabetical order so we're not "picking favorites" though of course we may have _personal_ favorites the page should not :)

File: ide-alternatives.md (hunk 1 above)
Comment (hassila) [reply ide-alternatives.md]: Of course, fixed in 8639998

**Comment by hassila** [comment]
Fixed in 8639998, also put out LSP in separate section for nicer formatting.

File: ide-alternatives.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+[Emacs plugin](https://github.com/swift-emacs/swift-mode)
+
+[VIM plugin](https://github.com/keith/swift.vim)
+
+[Visual Studio Code](https://code.visualstudio.com)
+
+[Visual Studio Code optional code completion plugin](https://marketplace.visualstudio.com/items?itemName=vknabel.vscode-swift-development-environment)
+
+[Xcode](https://developer.apple.com/xcode/ide/)
```
Comment (tomerd) [inline ide-alternatives.md:17]: one more idea, maybe add a line here that says something like:

`* Know about another IDE or IDE plugin? Please submit a PR to add them here!`

Comment (hassila) [reply ide-alternatives.md:17]: Sure, why not,  added one at the end!

**Review by tomerd — APPROVED** [review]

**Pull request merged**